source/configuration_manual/mail_crypt_plugin.rst

```diff
@@ -367,12 +367,12 @@ fix this later.
 
 FS driver syntax::
 
-  crypt:[maybe][algo=<s>:][set_prefix=<n>:][private_key_path=/path:][public_key_path=/path:][password=password:]<parent fs>``
+  crypt:[maybe:][algo=<s>:][set_prefix=<n>:][private_key_path=/path:][public_key_path=/path:][password=password:]<parent fs>``
 
-.. dovecotchanged:: 2.3.19
+.. dovecotchanged:: 2.4.0;3.0.0
```

Comment:
"ValueError: Version string should not contain ";": 2.4.0;3.0.0"

It does not like `;` here should be `2.4.0,3.0.0`

Reply:
oh, right